subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1,7 +1,7 @@
 /** @file
  *  @brief Bluetooth Object Transfer Client
  *
- * Copyright (c) 2020 - 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 - 2022 Nordic Semiconductor ASA
```

Comment:
This line appears at the top of the file but I cannot comment on it as it is too far away from changes:
```c
/* TODO: Temporarily here - clean up, and move alongside the Object Transfer Service */
```
I believe it can be removed now?